readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
Glad you guys raised all the questions. I had them in mind for a number of days, thats why I was waiting with adding function values.

In fact expecting an ISTF value in return is also not always ideal for performance cases, where a very simple value would result in additional function cals and ISTF parsing.

Suggestion: we can make ISTF value optional and also support regular CSS string values. If value is a string its a CSS value, if it is an array - it is an ISTF value, wdyt?